pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -245,10 +245,16 @@ self: super: builtins.intersectAttrs super {
       hledger = installHledgerManPages super.hledger;
       hledger-web = installHledgerManPages (hledgerWebTestFix super.hledger-web);
       hledger-ui = installHledgerManPages super.hledger-ui;
+
+      hledger_1_30_1 = installHledgerManPages
+        (doDistribute (super.hledger_1_30_1.override {
+          hledger-lib = self.hledger-lib_1_30;
```

Comment:
according to the header in configuration-nix.nix, it feels like this line should stay in configuration-common.nix. I have nothing against moving it though.

Reply:
In a way it will always be necessary, just for different versions. In any case it's just simpler having everything in one place…